modules/benchmark/R/create.BRR.R

```diff
@@ -9,48 +9,49 @@
 ##' 
 ##' @author Betsy Cowdery 
 
-create.BRR <- function(ensemble.id, workflow, con) {
+create.BRR <- function(ens_wf, con){
   
-  cnd1 <- workflow$hostname == fqdn()
-  cnd2 <- workflow$hostname == "test-pecan.bu.edu" & fqdn() == "pecan2.bu.edu"
-  cnd3 <- workflow$hostname == "pecan2.bu.edu" & fqdn() == "test-pecan.bu.edu"
-
-  if (cnd1 | cnd2 | cnd3) {
-    # If the ensemble run was done on localhost, turn into a BRR
+  cnd1 <- ens_wf$hostname == fqdn() 
+  cnd2 <- ens_wf$hostname == 'test-pecan.bu.edu' & fqdn() == 'pecan2.bu.edu'
+  cnd3 <- ens_wf$hostname == 'pecan2.bu.edu' & fqdn() == 'test-pecan.bu.edu'
```

Comment:
Here I'm just trying to find a way to say that test-pecan, and pecan2 are actually the "same" machine in that the files are accessible from either. 
But this is skirting around the issue that if one wants to turn an existing ensemble into a benchmark reference run and that ensemble was not run on localhost, getting all the information to populate the settings section of the reference run table id going to be difficult / may not be possible. This is because we may not be able to read the pecan.xml file on the remote host and the ensemble settings aren't saved anywhere else. This is something that @mdietze and I talked about earlier this week and maybe Mike can explain it better than I can.